redfish-core/include/event_service_manager.hpp

```diff
@@ -1085,23 +1085,6 @@ class EventServiceManager
             }
         }
     }
-    void sendBroadcastMsg(const std::string& broadcastMsg)
-    {
-        for (const auto& it : this->subscriptionsMap)
-        {
-            std::shared_ptr<Subscription> entry = it.second;
-            nlohmann::json msgJson;
-            msgJson["Timestamp"] =
```

Comment:
What about these fields Timestamp, Origin, Name?

Reply:
Don't need them anymore?

Reply:
This method will take care of the other fields

https://github.com/openbmc/bmcweb/blob/master/redfish-core/include/event_service_manager.hpp#L1014